docs/modules_management.rst

```diff
@@ -0,0 +1,194 @@
+installable through the ``pip install`` command. We can then upload the same file to pypi.org.
+
+.. code-block:: bash
+    pip install --upgrade pip setuptools wheel
+
+2. Create the package directory - in our case, we will call it ``airflow_operators``.
+
+.. code-block:: bash
+    mkdir airflow_operators
+
+3. Create the file ``__init__.py`` inside the package and add following code:
+
+.. code-block:: python
+
+    print("Hello from airflow_operators")
+
+When we import this package, it should print the above message.
+
+4. Create ``setup.py``:
+
+.. code-block:: python
+
+    import setuptools
+
+    setuptools.setup(
+        name='airflow_operators',
+    )
+
+5. Build the wheel:
+
+.. code-block:: bash
+
+    python setup.py bdist_wheel
+
+This will create a few directories in the project and the overall structure will look like following:
+
+.. code-block:: bash
+
+    .
+    ├── airflow_operators
+    │   ├── __init__.py
+    ├── airflow_operators.egg-info
+    │   ├── PKG-INFO
+    │   ├── SOURCES.txt
+    │   ├── dependency_links.txt
+    │   └── top_level.txt
+    ├── build
+    │   └── bdist.macosx-10.15-x86_64
+    ├── dist
+    │   └── airflow_operators-0.0.0-py3-none-any.whl
+    └── setup.py
+
+
+6. Install the .whl file using pip:
+
+.. code-block:: bash
+
+    pip install dist/airflow_operators-0.0.0-py3-none-any.whl
+
+7. The package is now ready to use!
+
+.. code-block:: pycon
+
+  >>> import airflow_operators
+  Hello from airflow_operators
+  >>>
+
+The package can be removed using pip command:
+
+.. code-block:: bash
+
+    pip uninstall airflow_operators
+
+For more details, see: https://packaging.python.org/tutorials/packaging-projects/
+
+
+Adding directories to the path
+------------------------------
+
+You can specify additional directories to be added to ``sys.path`` using the environment variable :envvar:`PYTHONPATH`.
+Start the python shell by providing the path to root of your project using the following command:
+
+.. code-block:: bash
+
+    PYTHONPATH=/home/arch/projects/airflow_operators python
+
+The ``sys.path`` variable will look like below:
+
+.. code-block:: pycon
+
+    >>> import sys
+    >>> from pprint import pprint
+    >>> pprint(sys.path)
+    ['',
+     '/home/arch/projects/airflow_operators'
+     '/home/arch/.pyenv/versions/3.7.4/lib/python37.zip',
+     '/home/arch/.pyenv/versions/3.7.4/lib/python3.7',
+     '/home/arch/.pyenv/versions/3.7.4/lib/python3.7/lib-dynload',
+     '/home/arch/venvs/airflow/lib/python3.7/site-packages']
+
+As we can see that our provided directory is now added to the path, let's try to import the package now:
+
+.. code-block:: pycon
+
+    >>> import airflow_operators
+    Hello from airflow_operators
+    >>>
+
+
+Additional modules in Airflow
+-----------------------------
+Airflow adds three additional directories to the ``sys.path``:
+
+- ``conf.get('core', 'dags_folder')``
```

Comment:
```suggestion
- DAG folder: It is configured with option ``[core]]`` in section ``dags_folder ``.
```
Could you use word description here, not code snippet? This may not be understandable for everyone.